metals/src/main/scala/scala/meta/internal/metals/ScalaVersions.scala

```diff
@@ -20,7 +20,7 @@ object ScalaVersions {
     }
 
   val isLatestScalaVersion: Set[String] =
-    Set(BuildInfo.scala213, BuildInfo.scala212, BuildInfo.scala211)
+    Set(BuildInfo.scala213, "2.12.9", BuildInfo.scala211)
```

Comment:
Is there a reason the server is not yet on 2.12.9? This message will get outdated once 2.12.10 is released

Reply:
> Is there a reason the server is not yet on 2.12.9? This message will get outdated once 2.12.10 is released

Just wanted to keep the impact of #867 smaller. I can follow up with a PR for moving the server to 2.12.9